lib/LMS.class.php

```diff
@@ -1448,10 +1448,10 @@ public function GetQueue($id)
         return $manager->GetQueue($id);
     }
 
-    public function GetQueueContents($ids, $order = 'createtime,desc', $state = NULL, $owner = 0, $catids = NULL, $removed = NULL)
+    public function GetQueueContents($ids, $order = 'createtime,desc', $state = NULL, $priority, $owner = 0, $catids = NULL, $removed = NULL, $netdevids = NULL, $netnodeids = NULL)
```

Comment:
Bardzo niedobra zmiana - wprowadzasz nowy parametr, który nie ma wartości domyślnej posiadanej przez wcześniejsze parametry. W efekcie nie będzie można również pominąć wcześniejszych parametrów.

Reply:
Czy to będzie linia 171-172 w pliku lib/LMSManagers/LMSHelpdeskManager.php tam należy zadeklarować domyślna wartość czy chodzi o jakieś inne miejsce?

Reply:
Czy tak jak poprawiłem jest OK?

Reply:
Powyżej nadal widzę bez wartości domyślnej - chyba, że patrzę nie w tym miejscu co trzeba?

Reply:
Czy chodzi o parametr "priority"? 
Jeśli tak to poprawiłem. Gapiłem się na tą funkcję dobre kilka minut zanim zczaiłem o co chodzi.